1.7.01.7.0 correspondence

csKyou opened the pull request:
· Modified to use Cell meta acquisition API
· Fixed the use of Box meta acquisition API

csKyou committed:
1.7.01.7.0 correspondence
· Modified to use Cell meta acquisition API
· Fixed the use of Box meta acquisition API

dixonsiu commented on src/html/js/common.js:
```diff
@@ -161,13 +183,21 @@ Common.getBoxUrlFromResponse = function(info) {
+        Common.accessData.cellName = boxObj.cell.name;
+        Common.accessData.boxName = boxObj.box.name;
+    }).fail(function(xmlObj) {
+        if (xmlObj.status == "200") {
```
In line 125, the condition is "not equal", isn't it "not equal" here, too?

dixonsiu commented on src/html/js/common.js:
```diff
@@ -177,7 +207,19 @@ Common.getUnitUrl = function() {
     var result = cellUrl;
     if (cellUrl.startsWith(Common.PERSONIUM_LOCALUNIT)) {
-        result = cellUrl.replace(Common.PERSONIUM_LOCALUNIT + "/", Common.getUnitUrl());
+        if (Common.path_based_cellurl_enabled) {
```
Not sure why you have to change it? Are you forcing all URL to use the new format?

csKyou committed:
1.7.0 correspondence
・Corrected judgment of return value of boxCell API was wrong
・Correction because the state judgment of path_based_cellurl_enabled was wrong
